spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/README.adoc

```diff
@@ -0,0 +1,54 @@
+= Spring Cloud GCP Secret Manager Sample Application
+
+This code sample demonstrates how to use the Spring Cloud GCP Secret Manager integration.
+
+To begin using Secret Manager support, first add the `spring-cloud-gcp-starter-secretmanager` dependency to the project:
+
+```
+<dependency>
+	<groupId>org.springframework.cloud</groupId>
+	<artifactId>spring-cloud-gcp-starter-secretmanager</artifactId>
+</dependency>
+```
+
+Adding the starter dependency will automatically cause your GCP Project's secrets to be injected into your application's environment at bootstrap time.
+
+This sample application defines a `@ConfigurationProperties` in order to access and use the secret values.
+
+```
+@ConfigurationProperties
+public class MyAppProperties {
+
+	@Value("${application-secret}")
+	private String applicationSecret;
+	...
+}
+```
+
+== Running the Sample
+
+1. Create a Google Cloud Platform project with https://cloud.google.com/billing/docs/how-to/modify-project#enable-billing[billing enabled], if you don't have one already.
+
+2. Enable the Secret Manager API from the "APIs & Services" menu of the Google Cloud Console.
```

Comment:
Can you show the gcloud command to do this?

Reply:
Done.